CHANGES.md

```diff
@@ -1,3 +1,18 @@
+Release 4.41 (release date: 2019-07-02)
+=======================================
+New Features
+------------------
+- Added HMAC key management for service accounts. This includes a new "hmac"
+  command to create/get/update/delete HMAC on service accounts. Currently,
+  your project must be whitelisted for use with the new BPO functionality.
+  This restriction will be lifted in the near future.
+
+ Bug Fixes
+------------------
+- Fixed bug where copying files > 100MiB from GCS to S3 hangs.
```

Comment:
Nit: Old behavior should generally be written in past tense.

s/hangs/would hang/